eng/pipelines/handlers.yml

```diff
@@ -197,7 +197,7 @@ stages:
                   skipAndroidImages: true
                   installDefaultAndroidApi: false
                   skipXcode: ${{ ne(BuildPlatform.name , 'macOS') }}
-                  ${{ if parameters.UseProvisionator }}:
+                  ${{ if or(parameters.UseProvisionator, variables['internalProvisioning']) }}:
```

Comment:
@mattleibow  I can t get this to work, the idea is we could use the variable or set as default on devdiv.. do you have any idea what I m missing  seems this is never true its always passing skipProvisioning: false